autopush/db.py

```diff
@@ -363,6 +379,7 @@ def fetch_notifications(self, uaid):
 
     @track_provisioned
     def save_notification(self, uaid, chid, version):
+        # type: (str, str, Optional[int]) -> bool
```

Comment:
I don't think it is, it str()'s it, so if it was None, then the later eq match would fail to compare afaik.